pkg/util/grpcclient/register.go

```diff
@@ -0,0 +1,66 @@
+package grpcclient
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/kubeedge/mappers-go/config"
+	dmiapi "github.com/kubeedge/mappers-go/pkg/apis/dmi/v1"
+	"github.com/kubeedge/mappers-go/pkg/common"
+	"google.golang.org/grpc"
```

Comment:
the order of the import packages should be optimized

Reply:
done